connect4.js

```diff
@@ -58,10 +58,15 @@ function findSpotForCol(x) {
   return 0;
 }
 
-/** placeInTable: update DOM to place piece into HTML table of board */
-
 function placeInTable(y, x) {
-  // TODO: make a div and insert into correct table cell
+  const piece = document.createElement('div');
+  piece.classList.add('piece');
+  piece.classList.add(`p${currPlayer}`);
+  piece.style.top = -50 * (y + 2);
```

Comment:
what is the logic here?

Reply:
As we discussed, I'm not sure about line 65, but I believe this is an artifact of when I was trying to solve an issue with placing the pieces, and tried several things, including plugging in part of the solution. After we talked, I went and removed line 65, and the functionality was not affected. I believe that the answer is that line 65 makes sense in the context of the solution file, but not in my app, so I should remove it.

Lines 62 - 64 are clear, I believe. they add `.piece` class to all pieces, and `p1` or `p2` for player1 and player2, respectively. I tried changing the values and also commenting out line 65, and found no difference in the functionality or styling.

Should I close this request and push a new one with line 65 removed, or merge and push a new PR with line 65 removed?

Reply:
Never mind. I just realized how to resolve this. I'll push a new commit with line 65 removed to this request.

Reply:
Actually, if I remove line 65 here, it will be overwritten when we merge in subsequent PRs. So I suppose I should do it at as the last PR in the stack, after all, right?

Reply:
I added commit [f71a8bb](https://github.com/XiaoA/connect-4-game/pull/8/commits/f71a8bb7175352ba1508b7f088a34c7fe6df16ad), which removes line 65, to PR 7. If you'd prefer that I create a separate PR for that, I can do that.